src/Control/Monad/Bayes/Sampler.hs

```diff
@@ -76,34 +88,27 @@ instance Applicative SamplerST where
 instance Monad SamplerST where
   (SamplerST x) >>= f = SamplerST $ x >>= runSamplerST . f
 
--- | Run the sampler with a supplied seed.
```

Comment:
Once we have figured out how to use monadic generators we should put this back in (with a constraint to only allow it to be used on RNGs which support (are instances of) `FrozenGen`.